### DISCUSSION TITLE
Unable to run pb2nc on casper

### DISCUSSION THREAD

**Discussion by tcram** [dtcenter/METplus#2478] [Existing Builds] [answered]
Hi,
it appears the intel math library libimf.so is no longer available on casper.  Can the casper build for the most recent version be updated?

```
casper35(43833)-240127213213> pb2nc /gpfs/csfs1/collections/rda/data/ds337.0/prepnr/2022/prepbufr.gdas.2022071006.nr /gpfs/csfs1/collections/rda/transfer/dsrqst/ZHONG725520/prepbufr.725520.20220710.t06z.nc /gpfs/csfs1/collections/rda/work/tcram/ds337.0/config/pb2nc/PB2NCConfig_725520 -v 2

Error Execute: pb2nc /gpfs/csfs1/collections/rda/data/ds337.0/prepnr/2022/prepbufr.gdas.2022071006.nr /gpfs/csfs1/collections/rda/transfer/dsrqst/ZHONG725520/prepbufr.725520.20220710.t06z.nc /gpfs/csfs1/collections/rda/work/tcram/ds337.0/config/pb2nc/PB2NCConfig_725520 -v 2

pb2nc: error while loading shared libraries: libcudart.so.11.0: cannot open shared object file: No such file or directory;
```
Thanks,
-- Tom

**Comment by jprestop** [comment]
@tcram Thanks for letting us know you're having trouble with MET on casper.  I will rebuild MET on casper and will follow up once the build is complete.

**Reply by jprestop** [reply]
@tcram Just a follow up - we started installing the MET software using a role account.  It looks like I don't have access to that role account on Casper yet, so I'll need to write to CISL and get access before I can rebuild the software on Casper.  I'll submit that request now.

**Comment by tcram** [comment]
Hi Julie,

Sounds good. Thanks for the update and quick response.

— Tom

**Comment by jprestop** [comment, marked as answer]
Hi @tcram.  I have re-installed MET and METplus on both Casper and Derecho.  You can find the appropriate modules to load on our METplus [Existing Builds](https://dtcenter.org/community-code/metplus/metplus-5-1-existing-builds) page under "NCAR machines" for the 5.1 coordinated release.  I hope to have METcalcpy, METdataio, and METplotpy installed there as well by early next week.  Please let us know if you have any questions or problems.

**Reply by tcram** [reply]
@jprestop Thanks!  I tested it out on casper and it's working for me.

**Reply by jprestop** [reply]
@tcram Great!  Thanks for letting us know!
